packages/core/helper-plugin/lib/src/components/GenericInput/index.js

```diff
@@ -226,7 +226,7 @@ const GenericInput = ({
           hint={hint}
           name={name}
           onChange={(date) => {
-            const formattedDate = date.toISOString();
+            const formattedDate = date?.toISOString();
```

Comment:
What do you think about adding a comment? I think this case might not be obvious.

Reply:
yes sure thanks gustav